sdks/python/apache_beam/pipeline.py

```diff
@@ -788,8 +788,11 @@ def is_side_input(tag):
       for si, pcoll in zip(result.transform.side_inputs, side_inputs):
         si.pvalue = pcoll
       result.side_inputs = tuple(result.transform.side_inputs)
-    result.parts = [
-        context.transforms.get_by_id(id) for id in proto.subtransforms]
+    result.parts = []
+    for id in proto.subtransforms:
```

Comment:
Please use a different variable name (maybe `transform_id`)?  Python has `id` as a built-in function.

Reply:
Done.